_posts/2021-09-27-solidity-0.8.8-release-announcement.md

```diff
@@ -101,14 +99,15 @@ inside the constructor but you cannot use them to initialize other
 immutable variables at the point of their declaration.
 
 
-### Import Paths
+### Import Include Paths
 
 We added a new option `--include-path` to `solc` that hopefully makes it much easier to compile
 more complicated projects using multiple libraries. You can use it to specify
```

Comment:
Oh, one more thing. We might also say that `--include-path` is meant to be an alternative to import remapping in simple cases (i.e. where they are not used to rename the file itself). For example Brownie currently uses remapping to resolve dependencies and I think that they could switch to this feature instead.

Reply:
Added another paragraph.